examples/index.md

```diff
@@ -325,54 +289,41 @@ HTTP/1.1 422 Unprocessable Entity
 Content-Type: application/vnd.api+json
 
 {
+  "jsonapi": { "version": "1.0" },
```

Comment:
so, I see you wrote `json-api` there even though the spec is `jsonapi`.  This is going to be a common source of error, that I expect clients will likely accepts both... I've doubled checked that it's `jsonapi` myself :)

The json-schema is

```json
    "jsonapi": {
      "description": "An object describing the server's implementation",
      "type": "object",
      "properties": {
        "version": {
          "type": "string"
        },
        "meta": {
          "$ref": "#/definitions/meta"
        }
      },
      "additionalProperties": false
    },
```